From b886c5f99dd5ccd07fc0ba8eefa9871c21b5de9c Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:25:21 -0400
Subject: [pci] Change pci_bus_region addresses to dma_addr_t

Message-id: <20140719042520.19438.46068.stgit@gir.stowe>
Patchwork-id: 85987
O-Subject: [RHEL7.1 PATCH 25/81] PCI: Change pci_bus_region addresses to dma_addr_t
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: 0a5ef7b914be91dd257ae4f35223dd822dd3703a

commit 0a5ef7b914be91dd257ae4f35223dd822dd3703a
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Sat Dec 21 08:39:47 2013 -0700

    PCI: Change pci_bus_region addresses to dma_addr_t

    Struct pci_bus_region contains bus addresses, which are type dma_addr_t,
    not resource_size_t.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/pci.h b/include/linux/pci.h
index ba9d304..c320ade 100644
--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -583,8 +583,8 @@ int raw_pci_write(unsigned int domain, unsigned int bus, unsigned int devfn,
     int reg, int len, u32 val);
 
 struct pci_bus_region {
- resource_size_t start;
- resource_size_t end;
+ dma_addr_t start;
+ dma_addr_t end;
 };
 
 struct pci_dynids {
-- 
1.7.1